osm/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/osm/WorkflowOSM.groovy

```diff
@@ -216,7 +223,7 @@ create {
                             if(!osmprocessing.execute(h2gis_datasource:h2gis_datasource,
```

Comment:
do we add areaBuildingTypeAndComposition and FloorAreaTypeAndComposition in the processing_parameters ?